From bd2e1c0b30ea21a0f7f3a8f711246a3d011d316a Mon Sep 17 00:00:00 2001
Message-Id: <bd2e1c0b30ea21a0f7f3a8f711246a3d011d316a.1368111913.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:53 +0200
Subject: [PATCH 19/65] char: rename qemu_chr_guest_close() ->
 qemu_chr_fe_close()

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <78cb7d644a04f685f54c21e26095271663e988f8.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50797
O-Subject: [RHEL6.5 qemu-kvm PATCH 19/65] char: rename qemu_chr_guest_close() -> qemu_chr_fe_close()
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 2817822dced78521ab7c532ea50959aa248ea8c0)

Signed-off-by: Amit Shah <amit.shah@redhat.com>

Conflicts:
	hw/virtio-console.c
	qemu-char.h

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 hw/virtio-console.c | 2 +-
 qemu-char.c         | 2 +-
 qemu-char.h         | 2 +-
 usb-redir.c         | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/virtio-console.c | 2 +-
 qemu-char.c         | 2 +-
 qemu-char.h         | 2 +-
 usb-redir.c         | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hw/virtio-console.c b/hw/virtio-console.c
index ec510f1..cc758ec 100644
--- a/hw/virtio-console.c
+++ b/hw/virtio-console.c
@@ -67,7 +67,7 @@ static void guest_close(VirtIOSerialPort *port)
     if (!vcon->chr) {
         return;
     }
-    return qemu_chr_guest_close(vcon->chr);
+    return qemu_chr_fe_close(vcon->chr);
 }
 
 /* Readiness of the guest to accept data on a port */
diff --git a/qemu-char.c b/qemu-char.c
index 6af1256..26c1580 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -2582,7 +2582,7 @@ void qemu_chr_fe_open(struct CharDriverState *chr)
     }
 }
 
-void qemu_chr_guest_close(struct CharDriverState *chr)
+void qemu_chr_fe_close(struct CharDriverState *chr)
 {
     if (chr->chr_guest_close) {
         chr->chr_guest_close(chr);
diff --git a/qemu-char.h b/qemu-char.h
index 7165dcb..6a5a3b1 100644
--- a/qemu-char.h
+++ b/qemu-char.h
@@ -81,7 +81,7 @@ CharDriverState *qemu_chr_open_opts(QemuOpts *opts,
                                     void (*init)(struct CharDriverState *s));
 CharDriverState *qemu_chr_open(const char *label, const char *filename, void (*init)(struct CharDriverState *s));
 void qemu_chr_fe_open(struct CharDriverState *chr);
-void qemu_chr_guest_close(struct CharDriverState *chr);
+void qemu_chr_fe_close(struct CharDriverState *chr);
 void qemu_chr_set_echo(struct CharDriverState *chr, bool echo);
 void qemu_chr_close(CharDriverState *chr);
 void qemu_chr_fe_printf(CharDriverState *s, const char *fmt, ...);
diff --git a/usb-redir.c b/usb-redir.c
index 59e3bab..1388b0b 100644
--- a/usb-redir.c
+++ b/usb-redir.c
@@ -1095,7 +1095,7 @@ static void usbredir_handle_destroy(USBDevice *udev)
 {
     USBRedirDevice *dev = DO_UPCAST(USBRedirDevice, dev, udev);
 
-    qemu_chr_guest_close(dev->cs);
+    qemu_chr_fe_close(dev->cs);
     qemu_chr_close(dev->cs);
     /* Note must be done after qemu_chr_close, as that causes a close event */
     qemu_bh_delete(dev->chardev_close_bh);
-- 
1.7.11.7

